Jenkinsfile

```diff
@@ -43,13 +45,14 @@ for(j = 0; j < jdks.size(); j++) {
                                     "MAVEN_OPTS=-Xmx1536m -Xms512m"], buildType, jdk) {
                             // Actually run Maven!
                             // -Dmaven.repo.local=… tells Maven to create a subdir in the temporary directory for the local Maven repository
-                            def mvnCmd = "mvn -Pdebug -U -Dset.changelist help:evaluate -Dexpression=changelist -Doutput=$changelistF clean install ${runTests ? '-Dmaven.test.failure.ignore' : '-DskipTests'} -V -B -ntp -Dmaven.repo.local=$m2repo -s settings-azure.xml -e"
+                            def mvnCmd = "mvn -Pdebug -U -Dset.changelist help:evaluate -Dexpression=changelist -Doutput=$changelistF clean install ${runTests ? '-Dmaven.test.failure.ignore' : '-DskipTests'} -V -B -Dmaven.repo.local=$m2repo -s settings-azure.xml -e"
 
                             if(isUnix()) {
-                                sh mvnCmd
+                                sh "$mvnCmd -ntp"
                                 sh 'git add . && git diff --exit-code HEAD'
                             } else {
-                                bat mvnCmd
+                                // TODO tool 'mvn' is still an old version without -ntp
```

Comment:
eek...  maybe `ToolInstallations` would help here (anyway still waiting for [INFRA-2375](https://issues.jenkins-ci.org/browse/INFRA-2375) / [INFRA-2080](https://issues.jenkins-ci.org/browse/INFRA-2080) )

Reply:
Yeah just waiting for 2080.